script/build

```diff
@@ -51,32 +74,25 @@ BIN_DIR="${ROOT_DIR}/bin"
 COMMAND=""
 
 # -p creates if the target doesn't exist, noop otherwise
-mkdir -pv "${BUILD_DIR}" && cd "${BUILD_DIR}"
```

Comment:
we still need to move into the directory

Reply:
That is handled by the presets now. No need to do it explicitly in the script -- at least not when a CMake Preset is in use.

Reply:
Okay - as long as it does not cause an issue for the other scripts for packaging. I see it's using:
```
$(sourceDir)/build/$(presetName)
```
for the binaryDir path. As long as we're using `cpack` that probably isn't an issue.

Still learning the CMake Presets functionality. Overall it's probably a good improvement, just takes some getting use to.